litestar/cli/_utils.py

```diff
@@ -98,6 +110,8 @@ def from_env(cls, app_path: str | None) -> LitestarEnv:
 
         port = getenv("LITESTAR_PORT")
         web_concurrency = getenv("WEB_CONCURRENCY")
+        uds = getenv("LITESTAR_UDS")
+        fd = getenv("LITESTAR_FD")
```

Comment:
Maybe more descriptive names here?

Reply:
I had it match the names from the uvicorn config, but i'm all for making them friendlier.

I can use `file_descriptor` and `unix_domain_socket` or something like that?

Reply:
question on this - are you referring to the environment variable name or the variables themselves?

Reply:
Here I was referring to the variables, but I think all the - variables, environment variables and CLI options) could be more descriptive. 

How about `--fd, --file-descriptor` and `--uds, --unix-domain-socket` for the CLI options?